index.html

```diff
@@ -177,10 +177,13 @@ <h3 class="sixth-sheet-developer-text">Skills</h3>
     <h2 class="seventh-sheet__contact-me">Contact me</h2>
     <p class="seventh-sheet__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
```

Comment:
- [ ] Please use the correct width for the contact section paragraph and make sure it matches the Design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166121366-5c36f9f9-29d0-40f9-9a5e-78f2d5d3fc95.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166121358-960f5dec-4917-4309-b596-cebfe5315033.png)